src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -247,29 +300,16 @@ public void onTraversalStart() {
 
         File workspaceDir = overwriteOrCreateWorkspace();
 
-        // If provided buffer size is less 1KB it is considered to be too small
-        // Either use default 16KB or any value larger than 10KB
```

Comment:
Preserve this recommendation on the right value for `vcfBufferSizePerSample` in the docs for the argument above.